telemetry/definitions/commonDefinitions.json

```diff
@@ -1217,12 +1223,24 @@
         },
         {
             "name": "sam_deploy",
-            "description": "Called when deploying a sam application",
+            "description": "Called when deploying a SAM application",
             "metadata": [{ "type": "result" }, { "type": "version", "required": false }]
         },
+        {
+            "name": "sam_sync",
+            "description": "Called when syncing a SAM application",
+            "metadata": [
+                { "type": "result" }, 
+                { "type": "duration" }, 
```

Comment:
This should span the time taken for a user to run through the workflow end-to-end. So UI + execution of `sam sync` itself.

Reply:
This is possible in VSC because we control the lifecycle of the process. If JB can't do this easily then we're better off omitting this field.